packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -213,7 +219,11 @@ export const ChatMessageComponentAsRichTextEditBox = (
     <ChatMyMessage
       attached={attached}
       root={{
-        className: chatMyMessageStyles.root
+        className: mergeStyles(
```

Comment:
ChatMyMessage is v9 component, mergeStyles is v8 styles api, chatMyMessageStyles is v9 styles. Please use mergeClasses in this case instead to use v9 styles api with v9 component